addons/web_editor/static/src/js/wysiwyg/widgets/color_palette.js

```diff
@@ -737,18 +805,17 @@ const ColorPaletteWidget = Widget.extend({
         this.el.querySelectorAll('.o_colorpicker_sections').forEach(el => {
             el.classList.toggle('d-none', el.dataset.colorTab !== buttonEl.dataset.target);
         });
-    },
+    }
     /**
      * Updates a gradient color from a selection in the color picker.
      *
      * @private
      * @param {Event} ev from gradient's colorpicker
      * @param {boolean} isPreview
      */
-    _updateGradientColor(ev, isPreview) {
-        ev.stopPropagation();
+    _updateGradientColor(params, isPreview) {
```

Comment:
Well I'd say give it a quick try to see if there are lots of work to adapt the calls with `isPreview` or not. A priopri I suspect there is only a few calls and that adapting it would be done in like 5 minutes, but if you notice that it's going to be more complicated it's a different story.

Did you have a look at other docs of the function that you have changed to see if you didn't forget to update it ?

Reply:
The problem is that `_onColorPickerSelectGradient` calls `_updateGradientColor(params)`.
If I refactor it to include `isPreview` in the params, it will mean that the code which call `_onColorPickerSelectGradient` will now be able to call `_updateGradientColor(params)` with `isPreview`.

I am not sure it makes sense API wise, that is why I wanted to avoid to take the decision.

Anyway, I've made a commit to merge `isPreview` into `params`. I'll squash it if you validate the changes.